src/screens/UserPortal/Campaigns/PledgeModal.module.css

```
@@ -34,20 +35,215 @@
+.compactInlineGroup :global(.form-control),
+.compactInlineGroup :global(select),
+.compactInlineGroup :global(input[type='number']) {
+  margin-left: auto;
+  align-self: center;
+  flex: 0 0 auto;
+  width: var(--pledge-compact-box-width);
+  min-width: var(--pledge-compact-box-width);
+  max-width: var(--pledge-compact-box-width);
+}
+
+.currencySection :global(.form-select),
```

Stylelint (17.5.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)